src/extension.ts

```diff
@@ -49,9 +50,15 @@ export async function activate(context: vscode.ExtensionContext) {
         const awsContextTrees = new AwsContextTreeCollection()
         const resourceFetcher = new DefaultResourceFetcher()
         const regionProvider = new DefaultRegionProvider(context, resourceFetcher)
+        const loginManager = new LoginManager(awsContext)
```

Comment:
Why not create a LoginManager in DefaultAWSContextCommands and loginWithMostRecentCredentials? Both of those took the awsContext before (DefaultAWSContextCommands still does) and can construct a LoginManager from that.

Reply:
The toolkit should only have one loginManager. The LoginManager contains the credentials cache, so if you instantiate more than one, each instance would have its own credentials cache.